tests/main/snap-sign/task.yaml

```diff
@@ -0,0 +1,29 @@
+summary: Run snap sign to sign a model assertion
+
+execute: |
+    echo "Creating a new key without a password"
+    expect -f successful_default.exp
```

Comment:
I believe this should be either a copy of, or point to `../create-key/successful_default.exp`